app/views/coronavirus/live_stream.html.erb

```diff
@@ -1,25 +1,49 @@
-<% content_for :title, "Update live stream" %>
-<% content_for :context, "Turn the Number 10 live stream on or off" %>
+<% content_for :title, "Add a new URL" %>
+<% content_for :context, "Make changes to the coronavirus landing page live stream" %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-three-quarters">
-    <!-- turn livestream on/off buttons -->
-    <div>
-      <% if @live_stream.state %>
-        <%= link_to "Turn it off", coronavirus_publish_live_stream_path(on: false), method: :post, class: "govuk-button govuk-button--warning" %>
-      <% else %>
-        <%= link_to "Turn it on", coronavirus_publish_live_stream_path(on: true), method: :post, class: "govuk-button" %>
+    <div class="govuk-form-group">
+      <%= form_with(url: "update_live_stream", method: "post", local: true) do %>
+        <%= label_tag(:url, "URL", class: "govuk-label" ) %>
```

Comment:
```suggestion
        <%= label_tag(:url, "YouTube livestream link", class: "govuk-label" ) %>
```

Reply:
I don't know where on the page we should put this, but thinking for if I were an uninitiated content designer, I'd like some more explanations of what this is and where to find the livestream link for today. A floating Google doc isn't necessarily the easiest thing to find!

Reply:
Yes totally agree with comments. 
- We discussed whether there should be checks on the validity of the youtube link, ie is it actually live, is it from the right user etc, and the decision was that it's out of scope. Ie the links are being added by humans and they can make sure it's taken from the right place. However there is a preview button to make sure that the link works as expected.

- I've been worrying about the publishing of another person's changes unexpectedly generally. Ie at the moment there seems to be multiple content designers making changes to copy and raising pr's at the same time. I _think_ the reason it's been ok is because the content PR's are only merged once they are happy to be made live, so if someone accidentally publishes the changes it's not an issue. But maybe we should make that a key part of the process, ie explicitly state that you should only merge a change to the yaml files if you are happy for the change to be live. 
- I thought I could add a progressive disclosure to the livestream partial with guidance on where to get the link from. I'll add it and re-request your reviews so you can see what you think.